docs/admin/projects.rst

```diff
@@ -602,8 +602,8 @@ Turns on vote casting for suggestions, see :ref:`voting`.
 
 .. _component-suggestion_autoaccept:
 
-Autoaccept suggestions
-++++++++++++++++++++++
+Automatically accept suggestions
+++++++++++++++++++++++++++++++++
```

Comment:
https://github.com/WeblateOrg/weblate/pull/10903 changes this in the code, docs can followup then.